### DISCUSSION TITLE
Need help for overflow related issue

### DISCUSSION THREAD

**Discussion by ankitj96** [vercel/next.js#72822] [Help]
### Summary

I created a website but when I open the website on mobile I am unable to scroll. On reaserch I found the issue is due to overflow styling. But when I remove the setting the website is broken. any help

**Comment by icyJoseph** [comment]
Link to site/repository?

**Comment by ankitj96** [comment]
I am testing the website locally as of now. So it amy not be fisible to share a link

**Comment by ankitj96** [comment]
Though I can  connect on meet if possible for you @icyJoseph

**Comment by icyJoseph** [comment]
Record a video and post it here. 

Can you share your global CSS?
